snapcraft.yaml

```diff
@@ -1544,21 +1554,24 @@ parts:
       set -eux
       craftctl default
       cd $CRAFT_STAGE/usr
-      find . -type f,l -exec rm -f $CRAFT_PART_INSTALL/usr/{} \;
-      find . -type f,l -name "*.so*" -exec bash -c "rm -f $CRAFT_PART_INSTALL/usr/{}*" \;
+      find . -type f,l -exec rm -fv $CRAFT_PART_INSTALL/usr/{} \;
+      find . -type f,l -name "*.so.*" -exec sh -c "rm -fv $CRAFT_PART_INSTALL/usr/{}*" \;
```

Comment:
`"*.so*"` -> `"*.so.*"`: Are we intentionally dropping files ending in `.so` here? Why?